configs/projects/pe-bolt-server-runtime-2019.8.x.rb

```diff
@@ -1,6 +1,6 @@
 project 'pe-bolt-server-runtime-2019.8.x' do |proj|
   proj.setting(:pe_version, '2019.8')
-  proj.setting(:rubygem_puppet_version, '6.25.0')
+  proj.setting(:rubygem_puppet_version, '7.13.1')
```

Comment:
2019.8.x tracks puppet 6. We do not want any puppet 7 there.

Reply:
Woops, that's my bad.

Reply:
👌  reverted that change

Reply:
We might as well bump to latest in the 6.x series while we are bumping gems though. So looks like 6.25.1?

Reply:
Bumped version to 6.25.1